iked/ikev2_pld.c

```diff
@@ -849,6 +849,9 @@ ikev2_pld_cert(struct iked *env, struct ikev2_payload *pld,
 		}
 	}
 
+	if (certid->id_buf) {
+		ibuf_free(certid->id_buf);
+	}
```

Comment:
can you repush without this part? I fixed this in openbsd and cherry picked it here

Reply:
done plus better handling of compile-options in pipeline